tensorflow/lite/micro/kernels/select.cc

```diff
@@ -164,25 +163,20 @@ TfLiteStatus SelectEval(TfLiteContext* context, TfLiteNode* node) {
       return kTfLiteError;                                           \
   }
 
-  if (data->has_low_rank_input_condition) {
-    MicroPrintf("Not yet implemented.");
-    return kTfLiteError;
-  } else if (data->requires_broadcast) {
+  if (data->requires_broadcast) {
     TF_LITE_SWITCH(input_x->type, BroadcastSelect5DSlow);
   } else {
     TF_LITE_SWITCH(input_x->type, Select);
```

Comment:
Let's remove these macros and instead directly make the function call.

The macros hurt readability and we have moved away from them in the other TFLM kernels as well.

Reply:
done